pixy/httpapi.go

```diff
@@ -0,0 +1,131 @@
+package pixy
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/gorilla/mux"
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/mailgun/manners"
+)
+
+const (
+	NetworkTCP  = "tcp"
+	NetworkUnix = "unix"
+
+	hContentLength = "Content-Length"
+
+	fTopicName  = "topic"
+	fRoutingKey = "key"
```

Comment:
Yeah, I'm clueless as to what `f` represents, considering both refer to constants in routes.  I think `h` corresponds to "header."  At the very least, I know that all the `f` constants are route-related, and the `h` constant has to do with headers.  So it's not amazingly descriptive, but it adds something.